### PR TITLE
First Draft of new Jupyter Book

### PR DESCRIPTION
**Pull request by kmpaul**
This first draft includes a "sanitized" version of the Xdev Whitepaper in the "about" section of the Jupyter Book and placeholders for the rest of the content.

### CODE REVIEW

**Comment by kmpaul** [comment]
@ncar-xdev/all: I fixed the weird image errors on the new site.  Please take a look at this site and provide your feedback.  I realize that reading the whole "About" section will take some time, but I would *really* appreciate your feedback in a PR review.  I think this site is important.  ***This site will be the source of all truth for Xdev going forward!***  So, if you think something should change, let me know in your review.

**Review by andersy005 — COMMENTED** [review]
@kmpaul, I don't know if you have seen this already but the links in the **main menu** appear to be broken. For instance, trying this sequence of navigation: `home` -> `handbook` -> `about` takes me to a non-existing page: https://61e04bdf9a78bb00ae7c5225--ncar-xdev.netlify.app/handbook/about. The same thing happens with this sequence: `home` -> `about` -> `handbook`

File: about/pangeo.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+Xarray builds upon both of these by providing a Pandas-like API for NumPy-like
+data arrays.  Hence, any NumPy-like array (or drop-in NumPy replacement) can be
+"handled natively" by Xarray.  The same is true for Dask, which provides
+distributed data representations for both DataFrames (i.e., Pandas-like) and
+DataArrays (i.e., NumPy-like), which are (mostly) drop-in replacements for
+Pandas DataFrames and NumPy ndarrays.  This means that Xarray data can be
+represented by Dask DataArrays (which themselves are composed of many NumPy
+ndarrays or CuPy ndarrays), Legate NumPy arrays, or any other "NumPy-like"
+array.  This is one of the primary reasons why Xarray is so strongly
+ncouraged by the Pangeo community; Xarray provides a common
```
Comment (andersy005) [inline about/pangeo.md]:
```suggestion
encouraged by the Pangeo community; Xarray provides a common
```

File: about/pangeo.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+kind of NumPy-like array (see the note on interoperability above), it would
+take little effort to build an Xarray object from any other kind of NumPy
+"drop-in" replacement, such as Legate NumPy arrays.  Hence, the determining
+factor in selecting the Array Structure is how you want the data represented
+_on your hardware_.
+
+
+### Compute Platform
+
+The Compute Platform can be either an HPC or cloud platform.  Deployment of
```
Comment (andersy005) [inline about/pangeo.md]:
```suggestion
The Compute Platform can be either an HPC or a cloud platform.  Deployment of
```

File: handbook/processes.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+This process document is meant as a guide for Xdev Project Team members on how to work on Xdev Projects and work together as a team.
+
+The processes described in this document are shaped by our team process values.
+
+### Team Process Values
+
+- It is preferable for the whole team to move forward slowly *together* than it is for one or two people to move forward quickly while others do nothing.
+- Allow the team to continue to make progress even when others are out or unavailable.
+- Be accommodating of team members schedules and let them read and respond to communication in their own time.
+- Use synchronous communication (e.g., video meetings/calls) for what it is best: real-time collaboration (e.g., pair programming).
```
Comment (andersy005) [inline handbook/processes.md]:
```suggestion
- Use synchronous communication (e.g., video meetings/calls) for what is best: real-time collaboration (e.g., pair programming).
```

**Review by andersy005 — COMMENTED** [review]
Thank you for putting this together, @kmpaul! I left a few/minor suggestions.

**Comment by kmpaul** [comment]
FYI:  And, if you want to know the reason for the weird image display errors the site was having yesterday, I figure it out and fixed it.  It was *very weird*, and I don't think I've ever seen such a weird error.  Read on if you want to learn what happened and how I fixed it.

<details>
<summary>Read on for more details...</summary>
So, if you remember yesterday, there were 2 problems that I was experiencing.  First, the Netlify preview link wasn't showing up in the PR, but fortunately that got fixed after deleting the old authentication token and creating a new one.  The second error was related to the logo and the images on the *branding* page not showing.  That error was very weird!

On my MacBook, when I build the Jupyter Book and looked at the site, everything looked fine.  All of the images were present, and it looked fine.  On the Netlify preview site, the images that weren't displaying were `xdev_logo_dark.svg` and `xdev_logo_light.svg` images (and the `xdev_logo_light.svg` was used for the logo, too).  And like I said, everything looked fine on my MacBook.

So, first I looked at the log of the site build (i.e., the `jupyter-book build .` step in the GitHub Workflow), and I noticed that there were warning messages saying that the logo image `images/xdev_logo_light.svg` didn't exist, and when generating the `branding` page there were warning that both `images/xdev_logo_light.svg` and `images/xdev_logo_dark.svg` were not found!  I was very confused, because I could verify that the images were in my repo on my laptop and that they were not listed as "untracked" by git!

So, I tried to reproduce the error in a Docker container.  I pulled down `ubuntu:latest` and ran the container.  I installed git and pulled down my PR branch (following the exact commands used in the GitHub Workflow log), and then I started installing and setting up conda so that I could install python and Sphinx and all the dependencies.  While I was doing all of that, I did an `ls images/` just to make sure that the image files actually got downloaded...  and they *were* downloaded but with *case differences!*  That is, the files that I visually found in the `images/` directory were called `Xdev_logo_light.svg` and `Xdev_logo_dark.svg`!  WTF!!!

I went back to my laptop to look at the files in the directory, and sure enough an `ls images/` on my laptop showed the files were not capitalized!  How can this happen?

So, I started to look on Google for anyone having a similar problem.  I think the actual Google search text was "`git linux file is capitalized and not on mac`".  (Horrible grammar, but it found what I needed.)  That search pulled up [this StackOverflow question](https://stackoverflow.com/questions/6899582/i-change-the-capitalization-of-a-directory-and-git-doesnt-seem-to-pick-up-on-it).  The second answer on the page suggested a simple `git mv` renaming fix:

```bash
git mv file1 tmp
git commit -m "Rename file1 temporarily"
git mv tmp file1
git commit -m "Renaming file1 to fix case"
```

So, on my laptop, I tried to do this and what I got was this error:

```bash
$ git mv xdev_logo_dark.svg tmp
fatal: not under version control, source=images/xdev_logo_dark.svg, destination=images/tmp
```

WTF?  I could actually *tab-complete* the filename `xdev_logo_d[tab]` on my laptop, and it found it!  But git was telling me that the file I tried to rename wasn't under version control!?!?

So, I tried using the capitalized version of the filename even though my laptop didn't recognize it:

```bash
$ git mv Xdev_logo_dark.svg tmp
```

**And that worked!**  I committed and then changed the filename back with `git mv tmp xdev_logo_dark.svg`, and that worked!  So, then I did the same thing with the other file and push the changes up to the PR branch.

That fixed the problem, and now the images are properly displayed on the Netlify preview.  

After a little bit of investigation, I also discovered that `git config --get core.ignorecase` returns `true` on MacOS and Windows (which are case-insensitive filesystems).  I *think* this means that if you rename a file by *only changing case* (e.g., `mv File file`), then git will not recognize a filename change, even if the filename displays differently with `ls`!  That means, after doing a `mv File file` (which works on a Mac, even though the filesystem is supposedly case-insensitive), git doesn't show any changes with a `git status` check.  So, it looks different on your laptop, and the build works, but the change is never pushed into the git repo.

*Really annoying!*

But anyway.  Problem fixed.
</details>

**Comment by kmpaul** [comment]
> @kmpaul, I don't know if you have seen this already but the links in the **main menu** appear to be broken. For instance, trying this sequence of navigation: `home` -> `handbook` -> `about` takes me to a non-existing page: https://61e04bdf9a78bb00ae7c5225--ncar-xdev.netlify.app/handbook/about. The same thing happens with this sequence: `home` -> `about` -> `handbook`

This should now be fixed.  (Waiting for the preview to rebuild.)

**Comment by kmpaul** [comment]
@ncar-xdev/all: Note that everything in the *Handbook* section and everything in the *Status* section really haven't been written, yet.  So, your input *what should be in those sections* would be appreciated, too.

**Review by andersy005 — APPROVED** [review]

**Review by mgrover1 — APPROVED** [review]
This looks great @kmpaul !

**Comment by kmpaul** [comment]
@mgrover1: We need to discuss whether you want to stay in Xdev and what you want your role to be after you move to Argonne.  I'm sure you will have other things you need to do, but Scott Collis is a great guy and I think he understands and appreciates what Xdev is trying to do.  So, he might be receptive to some level of "staying connected" with Xdev after you leave NCAR.  Personally, I don't have a problem with it and I'd love to keep connected with you.  So, let me know after you settle in after your move.

**Comment by kmpaul** [comment]
I am going to merge this PR and address the *Handbook* chapter with a different PR.

**Pull request merged**